services/app-api/src/emailer/emails/newPackageStateEmail.test.ts

```diff
@@ -315,27 +497,23 @@ test('includes expected data summary for a rate amendment submission State email
             ),
         })
     )
+    //Expect only have 1 rate names using regex to match name pattern specific to rate names.
+    expect(
+        template.bodyText?.match(
+            /-RATE-[\d]{8}-[\d]{8}-(?:CERTIFICATION|AMENDMENT)-[\d]{8}/g
+        )?.length
+    ).toBe(1)
     expect(template).toEqual(
         expect.objectContaining({
-            bodyText: expect.stringContaining(rateName),
+            bodyText: expect.stringContaining(
+                generateRateName(sub, sub.rateInfos[0], statePrograms)
+            ),
         })
     )
 })
 
 test('renders overall email as expected', async () => {
-    const sub: LockedHealthPlanFormDataType = {
-        ...mockContractAndRatesFormData(),
-        rateType: 'AMENDMENT',
-        contractDateStart: new Date('2021-01-01'),
-        contractDateEnd: new Date('2021-12-31'),
-        rateDateStart: new Date('2021-02-02'),
-        rateDateEnd: new Date('2021-11-31'),
-        rateDateCertified: new Date('2020-12-01'),
-        rateAmendmentInfo: {
-            effectiveDateStart: new Date('06/05/2021'),
-            effectiveDateEnd: new Date('12/31/2021'),
-        },
-    }
+    const sub: LockedHealthPlanFormDataType = mockContractOnlyFormData()
```

Comment:
Please make this  snapshot with a rate amendment rather than a new rate.

It is my fault that this is not clear in the test title. I very intentionally included a snapshot test with the full rate amendment to make sure we had some checks for that UI (which has some conditional logic). I've noticed that often we don't test rate amendments in the app because its more tedious and there more steps/ I always try to have at least one place where rate amendment workflow being tested. This was the place I included it for email snapshots.